### ISSUE TITLE
Arrays of strings cannot be initialized

### ISSUE THREAD

**Issue by hedgar2017** [vyperlang/vyper#2762]
### Version Information

* vyper Version (output of `vyper --version`): 0.3.2
* OS: linux
* Python Version (output of `python --version`): 3.10.2

### What's your issue about?

```
vyper.exceptions.StructureException: Multidimensional String arrays are not supported
  contract "tests/vyper/external/calldata/calldata_string_array.vy", function "f", line 2:9 
       1 @external
  ---> 2 def f(a: String[100][1])
  ```

**Comment by charles-cooper** [comment]
this is a documentation bug. we do not (currently) support arrays of strings. (related: https://github.com/vyperlang/vyper/issues/2733)

**Comment by charles-cooper** [comment]
dynarray sof strings were implemented in eb2becf15cd91fa8c2c18963426e609ed2580f0a
